docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -13,6 +13,64 @@ ms.devlang: azurecli
 
 # Azure CLI release notes
 
+## January 15, 2018
+
+Version 2.0.55
+### ACR
+* Allow force push a helm chart that doesn't exist.
+* Allow runtime operations without ARM requests.
+* Deprecating '--resource-group' parameter from 'az acr login', 'az acr repository', and 'az acr helm' commands.
+
+### ACS
+* Add support for new ACI regions
+
+### Appservice
+* webapp: az webapp config ssl upload, handling the scenario of uploading certificates on apps that are hosted on an ASE, where the ASE RG & App RG are different
+* webapp: az webapp up --sku defaults to P1V1 if the os is linux
+* webapp, functionapp: webapp deployment source config-zip, fixed to show the right error message when a deployment fails 
+* webapp: add az webapp ssh
+
+### Botservice
+* Add deployment status updates to `az bot create`
+
+### Configure
+* Add 'none' as a configurable output format.
+
+### CosmosDB
+* Added support for creating database with shared throughput
+
+### HDInsight
+* Add commands for managing applications
+* Add commands for managing script actions
+* Add commands for managing Operations Management Suite (OMS)
+* `hdinsight list-usage`: support to list regional usages
+* `hdinsight create`: remove default cluster type
+
+### Network
+* `traffic-manager profile create/update`: Add support for `--custom-headers` and `--status-code-ranges`. Add support for new routing types: Subnet and Multivalue.
+* `traffic-manager endpoint create/update`: Add support for `--custom-headers` and `--subnets`.
+* `ddos-protection update`: Fix issue where supplying `--vnets ""` to remove vnets caused a strack trace.
+
+### Role
+* `create-for-rbac`: deprecate `--password` for securer password auto-generated by CLI
+
+### Security
+* Initial Release
+
+### Storage
+* BREAKING CHANGE: `storage blob/file/container/share list`- Limit default number of results returned to be 5,000.
+  Use `--num-results *` for original behavior of returning all results.
+* `storage blob/file/container/share list`- log marker for next page to STDERR and expose `--marker` parameter.
+* Add `storage blob service-properties update` command with support for static websites.
+
+### VM
+* 'vm disk/unmanaged-disk' and 'vmss disk': parameter options are more consistent.
```

Comment:
Changes like this one need to start having better descriptions:
* Is this a breaking change? (seems like it.)
* What are the old parameter names, and what are the new parameter names?

(cc @tjprescott, @marstr)